index.html

```diff
@@ -7,15 +7,76 @@
   </head>
 
   <body>
-    <h1>Tic Tac Toe</h1>
-    <div id="tic-tac-toe"></div>
+    <h1>Let's Play!</h1>
+    <div id="tic-tac-toe">
+      <div class="row">
+        <div class="square" id="one"></div>
+        <div class="square" id="two"></div>
+        <div class="square" id="three"></div>
+      </div>
+      <div class="row">
+        <div class="square" id="four"></div>
+        <div class="square" id="five"></div>
+        <div class="square" id="six"></div>
+      </div>
+      <div class="row">
+        <div class="square" id="seven"></div>
+        <div class="square" id="eight"></div>
+        <div class="square" id="nine"></div>
+      </div>
+    </div>
+    <a href="file:///Users/lgranger/Documents/ADA/git/ProjectForks/js-tic-tac-toe/index.html" class="button">reset</a>
   </body>
   <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
   <script src="tic-tac-toe.js"></script>
   <script type="text/javascript">
     $(document).on('ready', function() {
-      console.log('create and begin the game here!');
+      var game = new TicTacToe();
```

Comment:
I would pull this logic out and put it with the other JS logic in your tic-tac-toe.js file so that all of the logic is located in the same place rather than jumping back and forth to find it in both locations.